services/managers/ips4_manager.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from django.conf import settings
+import requests
+import os
+
+logger = logging.getLogger(__name__)
+
+class Ips4Manager:
+    MEMBER_GROUP_ID = 3
+    MEMBER_ENDPOINT = "/api/core/members"
```

Comment:
`MEMBER_ENDPOINT = "/api/index.php?/core/members"`
if you dont add that you'll get a 404 response from ips4